File: GUI/src/components/Book.js
// React
import React from 'react';

export const Book = ({title, gender, author, releaseDate, frontPage}) => {
  return (
    <li className='list__item--books'>
    
    <img alt={title} src={ frontPage } className='item__image'></img>

      <div className='item__book'>
          <p>Título: {title}</p>
          <p>Género: {gender}</p>
          <p>Autor: {author}</p>
          <p>Año de publicación: {releaseDate}</p>
      </div>

    </li>
    
  )
}

File: data/book.py
class Book:

    def __init__(self, id, author, title, release_date, gender):
        self._id = id
        self._author = author
        self._title = title
        self._release_date = release_date
        self._gender = gender

    @property
    def id(self):
        return self._id

    @property
    def author(self):
        return self._author

    @property
    def title(self):
        return self._title

    @property
    def release_date(self):
        return self._release_date

    @property
    def gender(self):
        return self._gender
